scripts/pylib/twister/twisterlib.py

```diff
@@ -2217,7 +2220,10 @@ def run_cmake(self, args=[]):
         if out:
             with open(os.path.join(self.build_dir, self.log), "a") as log:
                 log_msg = out.decode(sys.getdefaultencoding())
```

Comment:
`.decode()` needs to be inside `try` block.

Reply:
This line is also as the same as line 683, the exception did not trigger here. Looking forward to your opinion, thanks!